### ISSUE TITLE
Change WIFI onboarding process

### ISSUE THREAD

**Issue by 3alpha** [dappnode/DNP_DAPPMANAGER#1051]
**Is your feature request related to a problem? Please describe.**

Currenly, DAppManager shows warning that default WIFI credentials are used. 

**Describe the solution you'd like**

During onboarding, make user change default WIFI password

**Additional context**

This may be an security issue since many users ignore warning that default credentials are used

**Comment by alexpeterson91** [comment]
Only thing to remember is that it's now the 3rd password you setup during current onboarding flow correct?

**Comment by 3alpha** [comment]
Yeah, that gets annoying. Maybe we should advise users to prepare pen and paper before starting. 😅 

Kidding aside, any ideas to simplyfy that?

**Comment by alexpeterson91** [comment]
Gotta brainstorm a bit on that since even the 2 password flow is not good for many users who don't read (or understand the difference) what each thing is or confused host and dappnode password etc. even last year I had people think the Wi-Fi password they set in the UI is the adminUI login password or host password and then take days to convince them otherwise lol. I think it's an easier fix for dappnode hardware could preset the password as is but append the last few digits of the SN of the box but that's a lot more work on the sales side. 
I'll work on ideas for simplifying the password setup workflow.
